release/test_release_gold.sh

```diff
@@ -52,4 +54,24 @@ function test_invoke_github_api_post {
 	assert_equals $(invoke_github_api_post "liferay-portal-ee/git/tags" "${tag_data}") "${LIFERAY_COMMON_EXIT_CODE_OK}" $(invoke_github_api_post "liferay-portal-ee/git/refs" "${ref_data}") "${LIFERAY_COMMON_EXIT_CODE_OK}"
 }
 
+function test_update_release_info_date {
```

Comment:
@kiwm, the commit message is still referencing a function which was removed from this pull, can you please adjust?

Thanks